course.js

```diff
@@ -60,7 +60,7 @@ CourseJS.Course = class Course extends CourseJS.Entry {
      * Gets course related information from this Entry.
      * @return {CourseInfo} This entry's courseInfo property.
      */
-    getInfo () {
+    getCourseInfo () {
```

Comment:
I figured out why I changed the getCourseInfo function on this class to getInfo, it was namely because I wanted to override the functionality of getInfo due to the fact that I changed CourseInfo to extend Info, this should be changed back